src/main/app/drafts/models/eligibility/eligibility.ts

```diff
@@ -0,0 +1,85 @@
+import { Serializable } from 'models/serializable'
+import { YesNoOption } from 'models/yesNoOption'
+import { IsIn } from 'class-validator'
+import { ValidationErrors } from 'features/validationErrors'
+import { ClaimValue } from 'drafts/models/eligibility/claimValue'
+import { ValidationGroups } from 'claim/helpers/eligibility/validationGroups'
+
+export class Eligibility implements Serializable<Eligibility> {
+
+  @IsIn(YesNoOption.all(), { message: ValidationErrors.YES_NO_REQUIRED, groups: [ValidationGroups.CLAIMANT_ADDRESS] })
+  claimantAddress?: YesNoOption
```

Comment:
Would you mind separating each property with an empty line to make this more readable?